src/test/java/Homework16.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.common.value.qual.StaticallyExecutable;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest{
+    @Test
+    public void Navigation(){
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id=hel']"));
+        registrationLink.click();
+        String registrationUrl ="https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(),registrationUrl);
```

Comment:
Please, add Homework16 in TestNG.xml to run tests

For example

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="Homework16"/>
        </classes>
    </test>
</suite>
```

Reply:
Ok added, hope it's right this time